scripts/test-check-packages.sh

```diff
@@ -45,7 +45,7 @@ cd -
 elastic-package stack up -d -v
 
 # Boot up the kind cluster
-kind create cluster
+kind create cluster --config $CURRENT_DIR/kind-config.yaml
```

Comment:
isn't it `pwd` instead of `CURRENT_DIR`?

Reply:
It looks like the path is still a problem. Could you try some other options like `dirname $0`, or `./scripts/kind-config.yaml`, etc.?

Reply:
Oops, yes, sorry for not being more explicit, in my suggestion I used `CURRENT_DIR` as a placeholder.

Reply:
No problem!